### DISCUSSION TITLE
Start/End time of pcap

### DISCUSSION THREAD

**Discussion by lev99999** [chmorgan/sharppcap#439] [General]
Hello,

How can I get the start time and end time of the pcap file? (Version 6)

(I'm looking for something equivalent to this command in Wireshark: capinfos -a -e <pcap name>)

Thank You

**Comment by kayoub5** [comment]
Timestamp of first packet, and timestamp of last packet, there is no metadata in the pcap file.

**Reply by lev99999** [reply]
ok. so how can I get the timestamp of the first package, and the last one?
Thank you!

**Reply by kayoub5** [reply]
iterate the packets in the file (check examples for that), each packet have a timestamp property.

**Reply by lev99999** [reply]
OK. Thank you!
